Test in basis points

brianmcmichael opened the pull request:
Updates mainnet test suite to test in basis points instead of `% * 1000`

kmbarry1 commented on src/DssSpell.t.sol:
```diff
@@ -490,12 +490,12 @@ contract DssSpellTest is DSTest, DSMath {
         // 1000000073014496989316680335
         assertTrue(duty >= RAY && duty < 1000000073014496989316680335);  // gt 0 and lt 1000%
-        assertTrue(diffCalc(expectedRate(values.collaterals[ilk].pct), yearlyYield(rates.rates(values.collaterals[ilk].pct / 10))) <= TOLERANCE);
+        assertTrue(diffCalc(expectedRate(values.collaterals[ilk].pct * 10), yearlyYield(rates.rates(values.collaterals[ilk].pct))) <= TOLERANCE);
```
Would be great if we could use basis points all the way down, but not a big deal if it's inconvenient for some reason.

kmbarry1 replied:
Same comment as on the other PR:

Sorry, looks like I didn't communicate clearly enough--I just meant that it was very inelegent to be multiplying pct by 10 here, so it would be nice if expectedRate could just expect a basis point argument, I didn't mean to use basis points to express all values.

brianmcmichael replied:
Pushed a change to refactor this.

kmbarry1 reviewed (approved):
LGTM

brianmcmichael commented:
Updated to use basis points for all values, updated all values to use Dai units.

This should eliminate many potential copy/paste errors when values are getting updated weekly and values are easier on human auditing.

kmbarry1 reviewed (approved):
LGTM
